frontend/src/scenes/sessionRecordings/SessionRecordingsTable.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { Link } from 'lib/components/Link'
+import { humanFriendlyDuration, humanFriendlyDetailedTime } from '~/lib/utils'
+import { SessionRecordingType } from '~/types'
+import { Table } from 'antd'
+import { sessionRecordingsTableLogic } from './sessionRecordingsTableLogic'
+import { PlayCircleOutlined } from '@ant-design/icons'
+import { useIsTableScrolling } from 'lib/components/Table/utils'
+import { SessionPlayerDrawer } from './SessionPlayerDrawer'
+
+interface SessionRecordingsTableProps {
+    personIds?: string[]
+    isPersonPage?: boolean
+}
+
+export function SessionRecordingsTable({ personIds, isPersonPage = false }: SessionRecordingsTableProps): JSX.Element {
+    const sessionRecordingsTableLogicInstance = sessionRecordingsTableLogic({ personIds })
+    const { sessionRecordings, sessionRecordingsLoading, sessionRecordingId } = useValues(
+        sessionRecordingsTableLogicInstance
+    )
+    const { setSessionRecordingId } = useActions(sessionRecordingsTableLogicInstance)
+    const { tableScrollX } = useIsTableScrolling('lg')
+
+    const columns = [
```

Comment:
I might've totally missed a mockup somewhere for this—did we decide to make the sessions recording table look different from today's sessions table?

<img width="1620" alt="Screen Shot 2021-09-24 at 10 43 08 AM" src="https://user-images.githubusercontent.com/13460330/134718168-c8f0f6f5-edad-4667-8d63-63f1119ca0fb.png">

<img width="1334" alt="Screen Shot 2021-09-24 at 10 40 50 AM" src="https://user-images.githubusercontent.com/13460330/134717967-04ac6df5-e954-4227-b1ca-800ec8fc7233.png">

Reply:
Really good question. The existing sessions table had  pretty different functionality + information than the new session recordings table - hence the different design.

The old one would:
* Expand/collapse to show the events within the session when the row was clicked. 
* A single session could relate to multiple session recordings, and the 'watch sessions' button would open an overlay that showed multiple session recordings that a user could click into.

But in session recordings, a single row relates to a single session recording, and the events are no longer associated with it. So the new behavior is that clicking on a row opens the session recording.

I definitely agree that the current table seems a bit sparse, and I want to bring in more information like start/end point back in as we build it out more.